### ISSUE TITLE
Text is mangled when generating PDF in Chrome

### ISSUE THREAD

**Issue by Devon-Dickson** [telerik/kendo-react#787]
## I'm submitting a...

* Bug report 

## Current behavior

Some text in the generated PDF has been mangled. 
<img width="483" alt="image" src="https://user-images.githubusercontent.com/34903992/99689019-72687580-2a54-11eb-8591-34807468532a.png">

## Expected behavior

The text should render appropriately. 
<img width="405" alt="image" src="https://user-images.githubusercontent.com/34903992/99689086-87dd9f80-2a54-11eb-95c2-3bf6aa5c3389.png">

## What is the motivation or use case for changing the behavior?

Fix this bug

## Environment

Package versions:

```
@progress/kendo-react-pdf@2.11.0
```

Browser:

- Chrome (desktop) version 86 and 87 - Broken
- Chrome (desktop) version 80 - Works
- Firefox version 84 - Works

System:
- Node version: 8.16.1 NPM version: 6.4.1  
- Platform:  Mac, Linux, Windows

It looks like this is happening when we are truncating text that is too large to fit in it's container.

**Comment by Devon-Dickson** [comment]
I've also managed to reproduce this issue in stackblitz
https://stackblitz.com/edit/react-dnd3zv?file=app%2Fmain.jsx

**Comment by leahcnels** [comment]
Based on a ticket that came up in planning mtg, we appear to be having this issue as well (version 3.18.0) I have not researched it yet but can provide more info when I do. It is a high priority issue for a client, so it would be good to know if it'll be looked into soon

**Comment by laurachelms** [comment]
Our client first reported this issue to us as having occurred on Safari, so I can confirm this is most commonly reproducible for us on Safari. Especially if the line ends in a special character like **;**, **-**, or **&**  it seems result in the next line laying on top of the previous line. Not necessarily consistent, but if the line is garbled it has ended in one of the characters listed above.

<img width="595" alt="Screen Shot 2020-11-20 at 2 57 54 PM" src="https://user-images.githubusercontent.com/32681691/99857789-e08a6680-2b40-11eb-8e20-cad47f0fbda0.png">
<img width="514" alt="Screen Shot 2020-11-20 at 2 58 01 PM" src="https://user-images.githubusercontent.com/32681691/99857791-e1bb9380-2b40-11eb-9ad6-bc66017cd509.png">

**Comment by GeoMarkou** [comment]
Hi is this on the roadmap for being resolved? We're experiencing this issue in our application.

**Comment by Xizario** [comment]
CC @telerik/kendo-dataviz

**Comment by simonssspirit** [comment]
reported again in 1539396 raising the priority

**Comment by pablorodrigovieira** [comment]
@simonssspirit please find an example within the link: 
https://stackblitz.com/edit/react-aurgvh-yktg4o?file=app%2Fmain.jsx

It's using the latest version and it breaks where there is a hyphen when generating the PDF in Safari, Version 15.0 (16612.1.29.41.4, 16612).

**Comment by tsvetomir** [comment]
Thanks for the additional sample.

The problem is caused by a mismatch between the screen and PDF font metrics.
Bootstrap sets the font family and font size via this rule:
```css
body {
  margin: 0;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5;
  color: #212529;
  text-align: left;
  background-color: #fff;
}
```

Since these fonts are not available to the export module, the text dimensions are incorrect in the exported file. The solution is to choose a font that's also available that's available for embedding. See [Unicode and Embedded Fonts](https://www.telerik.com/kendo-react-ui/components/drawing/pdf-output/embedded-fonts/).

I've [updated the demo](https://stackblitz.com/edit/react-aurgvh-2umtwz?file=index.html) to use OpenSans and to embed it in the PDF document. With this change, the text layout is correct in the exported document:

<img width="747" alt="image" src="https://user-images.githubusercontent.com/247917/138046324-94ba7089-f499-46f6-b640-1c90fbbb2ea2.png">

**Comment by pablorodrigovieira** [comment]
@tsvetomir thanks for sending that updated demo. 

It does work with paragraphs, however, I've created a couple more from your demo using a table and a Grid component but they don't seem to work when there are more hyphens.

[Grid example](https://stackblitz.com/edit/react-aurgvh-5zhcmz?file=app%2Fmain.jsx)
![image](https://user-images.githubusercontent.com/26431449/138188934-9378c1e0-48ef-47f4-8db2-649751181161.png)

[Table example](https://stackblitz.com/edit/react-aurgvh-qjal48?file=app%2Fmain.jsx)

![image](https://user-images.githubusercontent.com/26431449/138188896-6d1557de-55db-48a2-a5f7-77642dc82653.png)

do you have any demo or tips to fix within those scenarios?

Thanks in advance.

**Comment by tsvetomir** [comment]
@pablorodrigovieira apologies for the huge delay here. The good news is that we finally identified the cause for this bug and a fix is on its way.

**Comment by tsvetomir** [comment]
The latest development build of @progress/kendo-drawing contains a fix for this issue, see [updated demo](https://stackblitz.com/edit/react-aurgvh-75mmek?file=app%2Fmain.jsx).

Let me know if you experience any other issues related to the PDF export with the "dev" version. An official v1.7.2 release will follow soon.

**Comment by tsvetomir** [comment]
Fixed in v1.16.2
